### PR TITLE
RHELPLAN-41572 - Logging - Convert each subrole to a pair of task/var files

### PR DESCRIPTION
**Pull request by nhosoi**
Moved from https://github.com/nhosoi/logging/pull/8 to linux-system-roles/logging pull request.

### CODE REVIEW

File: design_docs/rsyslog_input_output_roles.md [hunk 1]
```diff
@@ -27,9 +27,10 @@ The remote logs are handed to \*.remote via remote ruleset.  Then, the logs are
 
 ### viaq-k8s
 
-## Output Roles
+## Output Tasks
 
 ### elasticsearch
+- `omelasticsearch` - configure outputs to elastchsearch.  See also [tests_files_elasticsearch.yml](../tests/tests__files_elasticsearch.yml) for an example.
```
Comment (richm) [inline design_docs/rsyslog_input_output_roles.md:33]: `elastchsearch` -> `elasticsearch`

File: design_docs/logging_subsystem.md [hunk 2]
```diff
@@ -3,136 +3,131 @@
...
-[12]: tcp or udp.  Default to tcp.<br>
-[13]: fqdn or IP address of the target host.<br>
-[14]: port number.  Default to 514<br>
+[1]: `logging_outputs` are implemented in logging/roles/rsyslog/tasks/outputs/ and logging/roles/rsyslog/vars/outputs/.<br>
+[2]: `output_type` is one of [elasticsearch, files, forwards]; the type matches the directory name in the `outputs` directory.<br>
+[3]: `logging_inputs` are implemented in ./logging/roles/rsyslog/tasks/inputs/ and ./logging/roles/rsyslog/vars/inputs/.<br>
+[4]: `input_type is one of [basics, files, ovirt, viaq, viaq-k8s]; the type matches the directory name in the `inputs` directory.<br>
+[5]: logging_flows are used in the inputs (inputs/{basics,files,etc.}) to set the `call output_ruleset`.
+[6]: inputs contains the list of unique_input_name(s).
+[7]:  outputs contains the list of unique_output_name(s).
```
Comment (richm) [inline design_docs/logging_subsystem.md:133]: space

File: roles/rsyslog/README.md [hunk 3]
```diff
@@ -298,8 +213,21 @@ Common sub-variables
 - `rsyslog_system_log_dir`: System log directory.  Default to '/var/log'.
 - `rsyslog_work_dir`: Working directory.  Default to '/var/lib/rsyslog'.
 
-Files and Forwards output_role sub-variables
+Elasitcsearc, Files and Forwards outputs sub-variables
 ----------------------------------
```
Comment (richm) [inline roles/rsyslog/README.md:217]: Elasticsearch

**Comment by pcahyna** [comment]
The "fixing typos" commit should be squashed together with the commit that introduced those typos. At least before you merge.

**Comment by nhosoi** [comment]
Thank you for your reviews, @richm, @pcahyna!  Yes, I'm going to squash the 4 commits into one.

**Pull request merged**